sdk/mgmtcommon/Auth/Az.Auth/Az.Authentication/ApplicationTokenProvider.cs

```diff
@@ -444,6 +444,33 @@ public static async Task<ServiceClientCredentials> LoginSilentAsync(string domai
 #endif
         }
 
+#if !net452
```

Comment:
Is this formatting change and rearranging of regions?
Or am I missing the change here?

Reply:
Sorry I was not able to figure out the diff, now I can :)
There is no harm in adding an overload, but you can still achieve the same using an existing overload on line 573 (new up AuthenticationProvider with your certificate and the additional flag)
The only downside to taking this change is to release this package and then update dependent packages and release them as well.
And if you can use the overload on line 573, we can avoid releases of these packages.
I will let @AlexGhiondea decide this.